README.md

```diff
@@ -1,10 +1,20 @@
-# Eligibility rules service exemplar - Research project
+# Eligibility rules service - Phase 2 (research/discovery)
```

Comment:
Two things here:

1) I feel like the word "exemplar" is not super plain-language and could possible cause folks to view this as more of an academic exercise than it actually is...? *I* know what you mean by it, but I worry others won't.

2) I'm wondering if we should actually treat this repo as the single home for all of this work around the rules service moving forward, rather than just the home for *this phase,*  so I'm wondering if we want to ditch "research" from the repo title, and for the readme, just indicate what phase the project is in?